api/resid/gvk.go

```diff
@@ -193,7 +193,7 @@ var notNamespaceableKinds = []string{
 // IsNamespaceableKind returns true if x is a namespaceable Gvk
 // Implements https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#not-all-objects-are-in-a-namespace
 func (x Gvk) IsNamespaceableKind() bool {
```

Comment:
What if we make a new utility function not on Gvk structure and this function just calls that one?
```
func (x Gvk) IsNamespaceableKind() bool {
     return util.IsNamespaceableKind(x.Kind)
}
```
Optionally also move NotNamespaceableKinds into utility package.

Reply:
Let us wait until https://github.com/kubernetes-sigs/kustomize/pull/2859 is merged. Then I will fix this PR.